tests/php/Core/ObjectTest.php

```diff
@@ -538,4 +540,26 @@ public function testInjectedExtensions()
         $this->assertInstanceOf(TestExtension::class, $object->getExtensionInstance($mockClass));
         $this->assertInstanceOf($mockClass, $object->getExtensionInstance($mockClass));
     }
+
+    /**
+     * @expectedException InvalidArgumentException
+     */
+    public function testExtensionsAppliedViaANonExantServiceAsAControl()
```

Comment:
```diff
- Exant
- Existant
```

Reply:
It's a word :)

Reply:
What benefit do you add by using it over "existant", which anyone would understand?

Reply:
It was shorter :P

Reply:
We have [some guidelines for writing style](https://docs.silverstripe.org/en/4/contributing/documentation/#writing-style), one of which is "Use simple language and words. Avoid uncommon jargon and overly long words."

As a native English speaker I don't know what "exant" means, nor can I find a definition for it (should it be "extant"? I still don't know what that means without looking it up). I can imagine this writing style being even harder to understand for a non-native English speaker =)